ship.config.js

```diff
@@ -23,7 +19,8 @@ module.exports = {
     );
 
     return {
-      // This is used for shouldPrepare
+      // This version number is used for shouldPrepare.
+      // for it to be useful, we need to chang shipjs to accept an array of nextVersions
```

Comment:
Should we add a `@TODO`?

Reply:
the todo is more in shipjs than here, so not sure if it's here it should be marked? can add it if you think it's useful of course

Reply:
Then that would be two of them:
- One in Ship.js for the change
- One here to express that we need to make a change once we update to Ship.js x.y.z

Can be done separately ofc.

Reply:
```suggestion
      // This version number is used for shouldPrepare.
      // for it to be useful, we need to chang shipjs to accept an array of nextVersions
      // @TODO: update this to packages.map(package => package.version) once it's supported in shipjs
      // see https://github.com/algolia/shipjs/issues/986
```